sdk/core/Azure.Core.TestFramework/src/ResourceGroupCleanupPolicy.cs

```diff
@@ -7,7 +7,7 @@
 using Azure.Core;
 using Azure.Core.Pipeline;
 
-namespace Azure.ResourceManager.TestFramework
+namespace Azure.Core.TestFramework
 {
     public class ResourceGroupCleanupPolicy : HttpPipelineSynchronousPolicy
```

Comment:
This implementation has to be thread-safe.

Reply:
Access to ResourceGroupsCreated is not syncronized.

Reply:
Why would this change, the namespace change, change the behavior of this class from its current behavior, or are you saying this is currently a problem with this class?

Reply:
There is currently a problem with the class.

Reply:
We are staying with parity with this PR